12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+map(chr, range(ord('a'), ord('z')+1))
```

Comment:
Yes, it is a reduce function and it is a good solution in general. It is just not called a list comprehension which would have the form of `[ ... ]`.